nixos/modules/virtualisation/containers.nix

```diff
@@ -518,6 +518,7 @@ in
         KillSignal = "WINCH";
 
         DevicePolicy = "closed";
+        DeviceAllow = "/dev/net/tun rw";
```

Comment:
I am not sure, what a good policy would be here.
One device people could also request in future is /dev/fuse.